### ISSUE TITLE
Double-check with compatible datasets for your works.

### ISSUE THREAD

**Issue by xtorker** [mauriceqch/pcc_geo_cnn_v2#3]
Hi Mr. Quach,

I have been benefited a lot from your excellent works including the first version of geo_cnn.
I am now preparing the extension experiments for my master thesis, and I want to include your geo_cnn_v2 and the attr_folding into the comparison.

I have roughly checked the codes, but I want to double-check with you in case I misunderstand something.
Below are my observations:
 1. For geo_cnn and geo_cnn_v2, the neural network model can only handle geometry when both training and testing. (not including the color mapping part done with finding NN point)
 2. For attr_folding, the whole pipeline will train a dedicated model for a single point cloud. The trained model can handle both geometry and color with separated python scripts.

If you can help me to verify my observations above, that will be a huge help.
Many thanks in advance.

**Comment by mauriceqch** [comment]
Hi @xtorker,

Happy to learn that my work was useful to you !

On your first point, you are correct as the model only compresses geometry. The attributes are transferred losslessly for visualization.

On your second point, the pipeline indeed trains a dedicated model for a single point cloud. More precisely, it proceeds as follows:

* The trained model learns a folded grid that reconstructs the original point cloud.
* We then build a mapping between the original geometry and the folded (2D) grid.
* With this mapping, the colors are transferred to the 2D grid and compressed/decompressed using image compression.
* On the decoder side, we can in theory retrain an identical model from the geometry and rebuild the mapping. 
* With this mapping, we can then transfer the decompressed colors on the 2D grid back to the original point cloud.

Please don't hesitate if you have any further questions.

**Comment by xtorker** [comment]
Hi @mauriceqch,

Thanks for your quick reply.

In your paper, you mentioned that attr_folding has to use the point cloud with geometry to train the 2D grid to the 3D point cloud folding model. In this case, do you mean that the attr_folding didn't handle the geometry compression, instead, it relies on other geometry compression codec?

Thanks again for your kindly reply.

**Comment by mauriceqch** [comment]
> Hi @mauriceqch,
> 
> Thanks for your quick reply.
> 
> In your paper, you mentioned that attr_folding has to use the point cloud with geometry to train the 2D grid to the 3D point cloud folding model. In this case, do you mean that the attr_folding didn't handle the geometry compression, instead, it relies on other geometry compression codec?
> 
> Thanks again for your kindly reply.

Yes, we assume the geometry was coded using another method and focus on attribute coding only.

**Comment by xtorker** [comment]
Thank you very much.
